src/main/resources/config/application.yml

```diff
@@ -10,7 +10,6 @@ security:
       tokenInfoUri: ${TOKENINFO_URL}
 
 management:
-  port: 7979
```

Comment:
Normal revocation port is randomly generated during tests, so there is no collision if the application is still running when a new HTTP server spawns. However, the Spring random port annotation does not work if the management port is fixed in the profile configuration, which leads to collisions. Why this is colliding in first place is something that I still don understand. But to be symetric to application port = random, I'd let management port be random in the tests too, and define the management port in the senza files.